src/plugins/discover/public/components/hits_counter/hits_counter.tsx

```diff
@@ -103,6 +117,18 @@ export const HitsCounter: React.FC<HitsCounterProps> = ({ mode, stateContainer }
           />
         </EuiFlexItem>
       )}
+      {hitsStatus === FetchStatus.ERROR && (
+        <EuiFlexItem grow={false}>
+          <EuiIconTip
+            type="warning"
+            color="warning"
+            size="s"
+            content={i18n.translate('discover.hitsCounter.hitCountWarningTooltip', {
+              defaultMessage: 'Results might be incomplete',
+            })}
```

Comment:
```suggestion
            content={i18n.translate('discover.hitsCounter.hitCountWarningTooltip', {
              defaultMessage: 'Results might be incomplete',
            })}
            iconProps={{ css: { display: 'block' } }}
```
Nit: changing the icon to `display: block` will prevent the flex item from filling vertically and improve the alignment a bit.

Reply:
Nice! Thanks for the suggestion!